File: Day1/odd_even.java
// execute command --> javap java.util.Scanner to see profile of particular class
import java.util.Scanner;    // Scanner class is used to read input in java
class Test{   // javap test
    public static void main(String[] args)
    {
        Scanner obj = new Scanner(System.in);
        //Scanner obj = new Scanner(String); // read data from string
        //Scanner obj = new Scanner(File); // read data from file
        //float n = obj.nextFloat(); // used to read float value
        
        int n = obj.nextInt(); //used to read int value.
        
        if(n>=0){
            if(n%2==0)
                System.out.println("even number"); // used to print output in java
            else
                System.out.println("odd number"); 
        }
        else{
            System.out.println("Invalid Number");
        }
    }
}

File: Day1/odd_even.c
#include<stdio.h>
int main(){
    int n;
    scanf("%d", &n);    // used to read inputs in C
    if(n>=0){
        if(n%2==0){
            printf("even number"); // used to print output in C
        }
        else{
            printf("odd number");
        }
    }
    else{
        printf("Invalid number");
    }
    return 1;
}